content/docs/reference-profiler.md

```diff
@@ -16,7 +16,7 @@ Its purpose is to help identify parts of an application that are slow and may be
 > To opt into production profiling, React provides a special production build with profiling enabled.
 > Read more about how to use this build at [fb.me/react-profiling](https://fb.me/react-profiling)
 
-## Usage
+## Usage {#usage}
```

Comment:
Revert this as you are not working on this page
`## Usage {#usage}` => `## Usage`